pkg/consts/consts.go

```diff
@@ -41,5 +41,7 @@ var (
 	NewBaseHashFunc = sha256.New
 
 	// DataCommitmentBlocksLimit is the limit to the number of blocks we can generate a data commitment for.
+	// NOTE: this is no longer used as we're moving towards Blobstream X. However, we're leaving it
+	// here for backwards compatibility purpose until it's removed in the next breaking release.
 	DataCommitmentBlocksLimit = 1000
```

Comment:
IIUC main already contains breaking changes slated for the next breaking release of celestia-core so I think this PR could remove this exported constant or create a follow-up issue to do so before v2 of this repo.

Reply:
This exported constant will be removed when we remove the blobstream module, but not sure when that will happen or how it will look

Reply:
[question] after this PR if the constant is not used in celestia-core and is only used in celestia-app, should we set it to `1000` in celestia-app and delete it here?

Reply:
sure